.ci.yaml

```diff
@@ -2346,6 +2383,30 @@ targets:
         ["devicelab", "android", "linux"]
       task_name: new_gallery_impeller__transition_perf
 
+  # Pixel 7 Pro, Impeller (Vulkan)
+  - name: Linux_pixel_7pro new_gallery_impeller__transition_perf
+    recipe: devicelab/devicelab_drone
+    bringup: true
+    presubmit: false
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab", "android", "linux", "pixel", "7pro"]
+      task_name: new_gallery_impeller__transition_perf
+
+  # Samsung A02, Impeller (OpenGL)
+  - name: Linux_samsung_a02 new_gallery_impeller__transition_perf
```

Comment:
Just a heads up that we do not have any samsung a02 in the prod pool yet. What is the plan to add capacity in prod before enabling these tasks there? @yusuf-goog

Reply:
We are not enabling them in prod yet. Once zac is happy, i'll move some over to prod.